pkg/aws/helpers.go

```diff
@@ -781,6 +781,26 @@ func GetResourceIdFromARN(stringARN string) (string, error) {
 	return parsedARN.Resource[index+1:], nil
 }
 
+func GetResourceIdFromOidcProviderARN(stringARN string) (string, error) {
```

Comment:
Can we re-use https://github.com/openshift/rosa/blob/master/pkg/aws/helpers.go#L764?

Reply:
Not really, that one takes the last index and the new one takes the first.

I'm going to refactor this based on the service of the parsed arn